src/fcmcts/fcmcts.jl

```diff
@@ -15,26 +15,26 @@ end
 mutable struct FCMCTSTree{S,A}
     # To track if state node in tree already
     # NOTE: We don't strictly need this at all if no tree reuse...
-    state_map::Dict{AbstractVector{S},Int64}
+    state_map::Dict{S,Int64}
```

Comment:
I see, so we're going to delegate the joint state / joint action aspect to the domain?

Reply:
Does beg the question of why we need FCMCTS at all. Can't we just use vanilla MCTS as the baseline then?

Reply:
I think we might be able to do away with `FCMCTS` need to check a few things with actions I think.